ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/templates/apicast-url-secret.yml.j2

```diff
@@ -4,5 +4,5 @@ metadata:
   name: {{ item }}
   namespace: {{ ocp4_workload_lpe_3scale_tenant_gw_project }}
 data:
-  AdminPortalURL: {{ ocp4_workload_lpe_3scale_portal_endpoint | b64encode}}
+  AdminPortalURL: {{ ocp4_workload_lpe_3scale_tenant_admin_access_endpoint | b64encode}}
 type: Opaque
```

Comment:
You may want a newline here - Travis might complain otherwise

Reply:
Done